src/app/layout.js

```diff
@@ -0,0 +1,74 @@
+import Image from "next/image";
+import Link from "next/link";
+import { SingleNav } from "../queries/navigations";
+import HygraphLogo from "../../public/svg/logo.svg";
+import HygraphMark from "../../public/svg/mark.svg";
+import NavList from "../components/NavList";
+import "./globals.css";
+
+export default function Layout({ children }) {
+  return (
+    <div className="px-6 max-w-3xl mx-auto">
+      <header className="py-10 relative">
+        <nav className="relative flex items-center justify-between sm:h-10 ">
+          <Link href="/" aria-label="Hygraph Next.js Blog Starter">
+            <Image
+              width={150}
+              src={HygraphLogo}
+              alt="Hygraph Logo"
+              className="hidden sm:block h-10"
+            />
+            <Image
+              src={HygraphMark}
+              alt="Hygraph Logo"
+              className="h-10 sm:hidden"
+            />
+          </Link>
+          <ul><NavList navId="main" /></ul>
+        </nav>
+      </header>
+      <main>{children}</main>
+      <footer className="bg-white mt-5 dark:bg-gray-900">
+        <div className="py-6 lg:py-8">
+          <div className="md:flex md:justify-between">
+            <div className="mb-6 md:mb-0">
+              <Link href="/" aria-label="Hygraph Next.js Blog Starter">
+                <Image
+                  width={150}
+                  src={HygraphLogo}
+                  alt="Hygraph Logo"
+                  className="hidden sm:block h-10"
+                />
+                <Image
```

Comment:
Both `width` and `height` attributes are required with Next Image component to avoid layout shift. I don't see any eslint errors/warnings though. It's probably worth checking the eslint setup.